### ISSUE TITLE
oelint.vars.dependsordered false positives with versioned dependencies

### ISSUE THREAD

**Issue by robertsemaskevic** [priv-kweihmann/oelint-adv#224]
Bitbake supports specifying versioned dependencies, i.e. `RDEPENDS_${PN} = "package (operator version)"` syntax. However, the linter or the parser seems to treat all characters as an actual package.

Minimum steps to reproduce:
```
$ bat dummy_%.bb
───────┬─────────────────────────────────
       │ File: dummy_%.bb
───────┼─────────────────────────────────
   1   │ SUMMARY = "abc"
   2   │ DESCRIPTION = "abc def"
   3   │ HOMEPAGE = "http://"
   4   │ LICENSE = ""
   5   │ 
   6   │ DEPENDS += "a b c d e f"
   7   │ 
   8   │ SRC_URI = ""
   9   │ 
  10   │ RDEPENDS_${PN} += "\
  11   │     a \
  12   │     b \
  13   │     c (>= 1.2.3) \
  14   │ "
  15   │ RDEPENDS_${PN} += "d (>= 4.5.6)"
  16   │ RDEPENDS_${PN} += "()..789=>e"
  17   │ RDEPENDS_${PN} += "f"
───────┴─────────────────────────────────

```
```
$ oelint-adv --quiet --color --noinfo --suppress=oelint.vars.homepageping dummy_%.bb
<...>/dummy_%.bb:10:warning:oelint.vars.dependsordered:'RDEPENDS_${PN}' entries should be ordered alphabetically
<...>/dummy_%.bb:15:warning:oelint.vars.duplicate:Item '(>=' was added multiple times to RDEPENDS_${PN}
```

Tested on python 3.6, oelint-adv 2.5.2, oelint-parser 1.1.1.
